nbgrader/plugins/latesubmission.py

```diff
@@ -0,0 +1,24 @@
+from textwrap import dedent
+from traitlets import Unicode
+
+from .base import BasePlugin
+
+
+class LateSubmissionPlugin(BasePlugin):
+
+    penalty_method = Unicode(
```

Comment:
There is an [enum trait type](http://traitlets.readthedocs.io/en/stable/trait_types.html#traitlets.Enum) which might be good to use here to ensure user typos throw an error.